### ISSUE TITLE
Keine Log Ausgabe im Log-Fenster / no log in log window

### ISSUE THREAD

**Issue by WilliLz** [ioBroker/ioBroker.javascript#1461]
**Describe the bug**  
Keine Log Ausgabe im Log-Fenster

**To Reproduce**  
kein Logging im Javascript-Adapter bei einem neu aufgesetztem System (14. Dez) 

System:
Proxmox VE 8.1.3, LXC 
  OS: Debian 12 
  Architektur: amd64
unpriviligierter Container: nein
nesting=1

bestehendes Template Debian 12 mit geklont (mit htop sudo net-tools nano unzip software-properties-common curl lsb-release apt-transport-https locate screen zip bzip2 ca-certificates gnupg wget)

sudo apt update && apt upgrade -y
curl -sLf https://iobroker.net/install.sh | sudo -E bash -
(mit Node.js: v18.19.0 und NPM: 10.2.3)

diverse Adapter installiert:
Used repository: stable
Adapter    "admin"        : 6.12.0   , installed 6.12.0
Adapter    "alpha-ess"    : 2.0.0    , installed 2.0.0
Adapter    "backitup"     : 2.8.7    , installed 2.8.7
Adapter    "devices"      : 1.1.5    , installed 1.1.5
Adapter    "discovery"    : 4.2.0    , installed 4.2.0
Adapter    "info"         : 2.0.0    , installed 2.0.0
Adapter    "javascript"   : 7.1.6    , installed 7.1.6
Controller "js-controller": 5.0.17   , installed 5.0.17
Adapter    "mqtt"         : 5.1.0    , installed 5.1.0
Adapter    "ping"         : 1.6.2    , installed 1.6.2

Probleme bei Ping-adapter beseitigt mit:
sudo nano /etc/sysctl.d/99-allow-ping.conf
    (added: net.ipv4.ping_group_range=1000 1001 (user, iobroker) hinzugefügt für Ping-Adapter)

Im javascript-Adapter funktionieren die Blocklys, aber im Log-Fenster ist KEINE Anzeige.
Man kann sich mit einem 2. Fenster behelfen, in dem man sich die Protokolle anzeigt - dort erfolgt die Ausgabe.
In der Log-Datei (cat /opt/iobroker/log/iobroker.current.log) sind keine Auffälligkeiten. Nur Infos

iobroker fix, Neustart des javascript-Adapters, neu laden des Controllers und Neustart des LXCs haben nichts am Verhalten geändert.

Beim upgrade des Controllers auf  einem andern LXC mit ähnlicher Konfiguration "js-controller": 5.0.16 auf Controller "js-controller": 5.0.17 trat der Fehler NICHT auf (ebenfalls Node.js: v18.19.0 und NPM: 10.2.3)

**Comment by WilliLz** [comment]
Update:
das Verhalten hat mir keine Ruhe gelassen. Ich konnte es weiter eingrenzen:
komplett neuen LXC mit neuem Container Image erstellt, iobroker installiert.
**Der Fehler lässt sich (bei mir) reproduzieren:**
Wenn man in den Basiseinstellungen - Systemeinstellung etwas beliebiges ändert und speichert und sichert es, wird die Seite grau. Man kann die Tabs anklicken, aber nichts mehr ändern. Durch ein reload der Seite kommt man wieder zurück und kann die Änderung sehen und den Dialog schließen.
Im Protokoll findet man folgenden Fehler:
`2023-12-15 21:29:01.323 - error: host.lxc-iob-testing _setDefaultAcl error on id "**META**" with acl "{"object":1636,"state":1636,"file":1632,"owner":"system.user.admin","ownerGroup":"system.group.administrator"}": Invalid ID: **META**`
Spätestens nach einem Neustart bleibt das Log-Fenster in JS-Adapter leer.

Es könnte also ein Bug woanders sein. (Admin, JS-Controller)

Liebe Grüße

**Comment by klein0r** [comment]
> _setDefaultAcl error on id "**META**"

Was ist denn das für eine Objekt-Id? Woher kommt der Name `**META**`

**Comment by ice987987** [comment]
> > _setDefaultAcl error on id "**META**"
> 
> Was ist denn das für eine Objekt-Id? Woher kommt der Name `**META**`

ich hab' die gleiche Fehlermeldung:

`2023-12-16 17:05:35.306 - error: host.iobroker-testsystem1 _setDefaultAcl error on id "**META**" with acl "{"object":1636,"state":1636,"file":1632,"owner":"system.user.admin","ownerGroup":"system.group.administrator"}": Invalid ID: **META**`

neu installiertes System auf Docker

**Comment by WilliLz** [comment]
@klein0r 
ich habe den Expertenmodus aktiviert und system.meta.uuid (leerer Ordner gefunden)
Wo das "META" her kommt, weiß ich nicht. Es steh nur einmal im log - wenn die Basiseinstellung das 1. Mal aufgerufen wird  und einfriert.
`*** nachgepostet, da ich auf mail geantwortet hatte ***`

**Comment by WilliLz** [comment]
Nach einem downgrade des JS-Controllers auf 5.0.16 arbeitete der JavaScriptadapter korrekt. Nach dem upgrade auf 5.0.17 dann auch. Es gab keine Probleme mehr!
Ich kann nicht entscheiden, wo der Report hin gehört.

**Comment by klein0r** [comment]
> Ich kann nicht entscheiden, wo der Report hin gehört.

Ich auch nicht, weil das das ganze Problem aktuell nicht nachvollziehen kann.

**Comment by WilliLz** [comment]
nein, ich habe REDIS nicht installiert - System war jungfräulich, habe dem ioBroker nur einen Anwendungsnamen gegeben,

**Comment by foxriver76** [comment]
ist das evtl ein Multihost mit unterschiedlichen Controller Versionen? `**META**` ist normalerweise ein Attribut in dem der Controller Daten (z. B. DB interne Protokollversion) vom Meta Namespace je DB File speichert. Noch nie gesehen, dass dieser Fehler auftritt. Villt auch eifnach nie gemeldet worden. Nur einmalig nach der Neuinstallation @ice987987?

**Comment by klein0r** [comment]
Bitte mit der aktuellen stable Version nochmal testen

**Comment by WilliLz** [comment]
der Fehler "Keine Log Ausgabe im Log-Fenster" besteht noch (lxc130) (vom 14.12.23)
 Invalid ID: **META**  trat nicht auf (update vom 15.12.23)
Proxmox VE 8.1.4, LXC, Adapter "javascript" : 7.1.6 , upgrade auf 7.8.0, Controller "js-controller": 5.0.17

**Comment by klein0r** [comment]
> Im javascript-Adapter funktionieren die Blocklys, aber im Log-Fenster ist KEINE Anzeige.

Wie ist denn das Loglevel für die JavaScript-Instanz definiert?

**Comment by WilliLz** [comment]
Das Loglevel steht auf Info.

Nach dem Ausführen des javaschripts > console.error('test');
kommt eine Anzeige im Logfenster, danach funktioniert auch mein Testscript (debug output >info).
 
Da ich mir blöd vor kam, habe ich den alten LXC neu geladen, der Fehler keine Ausgabe im Log-Fenster war wieder da. 
Nach anschließendem Aufruf des Javascripts   console.error('test'); funktionierte  die Ausgabe im Debugfenster wieder - auch mit dem Adapter "javascript" : 7.1.6 

Ich habe keine Idee was da quer war/ist, das Verhalten lässt sich ja replizieren.
(den Script werde ich mir mal bei Seite legen)

**Comment by klein0r** [comment]
Welchen Browser verwendest Du denn? Irgendwelche Fehler in der Entwickler-Konsole des Browsers?
